scripts/updateapilist.py

```diff
@@ -75,6 +82,39 @@ def __init__(self, repo: dict):
         self.release_level = repo["release_level"]
         self.distribution_name = repo["distribution_name"]
         self.issue_tracker = repo.get("issue_tracker")
+    
+    @property
+    def issue_tracker_component_id(self):
+        if not self.issue_tracker:
+            return None
+        match = re.search(r'(?:(?:\?|&)component=(\d+)|\bcomponentid:(\d+))', self.issue_tracker)
+        if match:
+            return match.group(1) or match.group(2)
+        return None
+    
+    @property
+    def issue_tracker_template_id(self):
```

Comment:
Is this used anywhere?

Reply:
In the create-issue URL, we should use the issue tracker if it was specified, but otherwise omit this query param.

Reply:
Do you mean to say that we should use the `template_id` if specified?

That's what my plan is. This function retrieves the template id from the `issue_tracker`. I will be incorporating that in the URL.